src/components/MainPage.js

```diff
@@ -20,23 +20,27 @@ class MainPage extends Component {
         }
     }
 
-    componentDidMount() {
-        API.get(`/todaymenu`, {headers: {'Authorization': localStorage.getItem('accessToken')}})
-          .then(res => {
-            console.log(res);
-            console.log(res.data);
+    //[CODE REVIEW] Przykład użycia składy async/await. Jest dużo wygodniejsza i czytelnieszja niż .then()
+    async fetchTodayMenu() {
```

Comment:
w javascripcie możesz używać składni async/await. Słowkiem "async" oznaczasz funkcję, która wykonuje kod asynchroniczny, czyli np. pobiera dane z serwera. "await" sprawia Ci się metoda zachowuje się synchronicznie, tzn. wszystko co jest poniżej w metodzie, czeka aż zapytanie się skończy. Dodatkowo opakowując to w try/catch możesz przechwycić błędy w catch. Składnia ta jest dużo, dużo czytelniejsza i unikasz callback hell